IdP/storage.go

```diff
@@ -60,13 +61,49 @@ func (s *storage) GetUser(ctx context.Context, userID string) (User, error) {
 // ChangePassword allows a user to change their password.
 func (s *storage) ChangePassword(ctx context.Context, userID string, password string) error {
 	// TODO(bimmlerd)
-	return fmt.Errorf("not implemented")
+	//hash password
+	h := sha1.New()
+	pwHash := h.Sum([]byte(password))
+
+	//does the user exist?
+	u := dbUser{}
+	row := s.db.QueryRowContext(ctx, `SELECT uid, firstname, lastname, email FROM users WHERE uid=?`, userID)
+	err := row.Scan(&u.uid, &u.firstname, &u.lastname, &u.email)
+	if err != nil {
+		if err != sql.ErrNoRows {
+			log.WithError(err).Error("Failed to query DB for user.")
+		}
+		return err
+	}
+	//change password.
+
+	row = s.db.QueryRowContext(ctx, `UPDATE users SET pwn = ? WHERE uid=?`, pwHash, userID)
```

Comment:
`pwn` to `pwd`

Reply:
Also, I think this should be `ExecContext` https://golang.org/pkg/database/sql/#DB.ExecContext since we don't expect results